libs/components/src/next/form/radio/vl-radio.component.ts

```diff
@@ -0,0 +1,122 @@
+import { CSSResult, html, PropertyDeclarations, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { live } from 'lit/directives/live.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { radioStyle } from '@domg/govflanders-style/component';
+import { baseStyle, resetStyle } from '@domg/govflanders-style/common';
+import { vlElementsStyle } from '@domg-wc/elements';
+import radioUigStyle from './vl-radio.component.uig-css';
+import { BaseLitElement } from '@domg-wc/common-utilities';
+
+export const RadioDefaults = {
+    id: '',
```

Comment:
Deze zie ik niet gebruikt worden in het component.

Reply:
Aangepast.